sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -337,6 +337,32 @@ object RapidsConf {
     .bytesConf(ByteUnit.BYTE)
     .createWithDefault(0)
 
+  val OFF_HEAP_LIMIT_ENABLED = conf("spark.rapids.memory.hostOffHeapLimit.enabled")
```

Comment:
Does it make sense to have a single, optional config for the off heap limit rather than a separate enable vs. size?  If it's not set, there is no limit.  If it's set, that's the limit and it's enabled.

Reply:
The goal is to eventually have enable go away after a short transition period.

The plan is to have this off by default while we develop the feature. Once the feature is done, then offHeapLimit is enabled by default, but the config is here so someone can disable it, if they run into problems during the transition. But the enable config would be deprecated with a clear release/date when it would be removed.